doc/source/contributing.md

```diff
@@ -0,0 +1,137 @@
+# Contributing Guide
+
+Thank you for deciding to contribute and help us improve Percona Server for MySQL documentation!
+
+We welcome contributors from all users and community. By contributing, you agree to the [Percona Community code of conduct](https://github.com/percona/community/blob/main/content/contribute/coc/).
```

Comment:
I suggest pointing to the live version of that document instead of the git repo:
```suggestion
We welcome contributors from all users and community. By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/).
```

Reply:
updated.